### DISCUSSION TITLE
Add Watching tab[withdrawn]

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#101] [Medium CIPs]
**UPDATE** As there has hardly been feedback I suspect this does not have much support. So i am withdrawing the CIP.

I posted this previously as an issue - so moving it here to get further feedback.

"Watching" feed tab

* Select box listing the "secondary" accounts, eg not the one you are logged in as right now
* Use it to view feeds from the other account, but reply, like, diamond with your main account
* This allows you to create simple twitter like Lists
* Selection is persisted through local storage
* Its live on tijn.club 

![121789123-40268d00-cbcb-11eb-9466-4fbfa38cf2a3](https://user-images.githubusercontent.com/69529928/127534898-9f28e081-4e6f-4acd-bc78-ca07965f7d38.gif)

**Comment by daniel-keyes** [comment]
This is a super interesting use case. Do you feel like it would get confusing to be logged into one account and reply from another? I can imagine a bunch of mistakes and or uncomfortable situations that arise from that.

What if the lists were built as a personalized feed that you define rather than from another account?

**Reply by tijno** [reply]
The purpose of the enhancement was to enable "list like" features without having to implement protocol changes. 

Feed back to date from users that have used it is that its not confusing.

**Comment by tijno** [comment]
As there has been no real response to this i will withdraw this CIP.
Hopefully someone can find a better way to implement list functionality that does not require a protocol change.
